### DISCUSSION TITLE
Projects beta: feature request - hiding columns

### DISCUSSION THREAD

**Discussion by lsl** [community/community#9228] [Projects and Issues] [answered]
I'm setting up a current sprint board and I'd like to hide some columns since they're never going to have anything in them in this view.

It would be good if there was a column option that lets you hide the column if its empty.

For example, we have a default status of "triage", its a full column on the "backlog" board view, but on the current sprint board view its usually going to be empty. It would be nice to have that column hidden while empty.

Similarly we have some other extra columns for planning but these only need to be shown on certain board views, having them show up empty on every board view isn't ideal.

**Comment by StoneCypher** [comment]
See #6906

**Comment by 2percentsilk** [comment, marked as answer]
We've added the ability to hide columns by updating your filter query (see https://github.blog/changelog/2022-09-02-the-new-github-issues-september-2nd-update/) 🥳

**Comment by jackonawalk** [comment]
Hi there, I'm using status filters already. It would be really useful if a view had an option like "hide empty columns automatically". This would be useful for grouping by status, sprint, etc.

**Reply by marcelcanusa** [reply]
I too would like to see this. We have a column that is only populated occasionally and we would only need it whenever there is something that has the associated status; when it's empty I would like it to hide itself automatically just like the "No status" column does.
